### ISSUE TITLE
Points in Lidar Point Cloud appear to be dublicated

### ISSUE THREAD

**Issue by plpalmer** [tudelft-iv/view-of-delft-dataset#72]
Hello,

i have noticed that Points in the Lidar Point Cloud seem to be dublicated. 
There always are tow points at exactly the same position.

This an be checked by 
```
x_nn = sklearn.neighbors.NearestNeighbors(n_neighbors=2, leaf_size=1, algorithm='kd_tree', metric='l2').fit(lidar_pc)
min_y_to_x = x_nn.kneighbors(lidar_pc)[0]
sumofdistances = min_y_to_x.sum()
```
the sumofdistances is always zero. Which means that for every point there are two points at the same position.

**Comment by andraspalffy** [comment]
Hello, 

I checked and it seems like you are right. I will have to see where the error is coming from. Luckily, it does not influence anything.  To get rid of the duplicates, just run:

`tolerance = 1e-6`
`point_cloud = np.unique(np.round(point_cloud / tolerance) * tolerance, axis=0)`

Thank you for bringing this to my attention!
